ENIGMAsystem/SHELL/Platforms/SDL/Event.cpp

```diff
@@ -208,11 +208,10 @@ void SDL_Event_Handler::windowFocusLost(const SDL_Event *event) {
 }
 
 void SDL_Event_Handler::windowResized(const SDL_Event *event) {
-  enigma_user::draw_clear(enigma_user::window_get_color());
+  graphics_remove_garbage(0,0,enigma_user::window_get_width(),enigma_user::window_get_height());
```

Comment:
I'm not sure this function needs any arguments as they should always be 0-width / 0-height. Might be better to just access those in the actual function.